node_exporter.go

```diff
@@ -110,6 +111,21 @@ func loadCollectors(list string) (map[string]collector.Collector, error) {
 		}
 		collectors[name] = c
 	}
+	for name, enabled := range collectorsEnabled {
```

Comment:
If I understand this correctly, it will only allow adding collectors via the per-collector flag, not removing them.